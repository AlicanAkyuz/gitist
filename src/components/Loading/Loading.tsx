import React, { FunctionComponent, Fragment } from 'react';

// type Props = JSX.IntrinsicElements['button'] & {
//   primary?: boolean;
//   secondary?: boolean;
// };

const Loading: FunctionComponent = () => (
  <Fragment>
    <header>Header</header>
    {/* <Routes /> */}
    <footer>Footer</footer>
  </Fragment>
);

export default Loading;
