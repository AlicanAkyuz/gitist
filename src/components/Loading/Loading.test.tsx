import React from 'react';
import { shallow } from 'enzyme';

import Loading from './Loading';

it('Loading corrently renders a loading message', () => {
  const wrapper = shallow(<Loading />);
  expect(wrapper).toMatchSnapshot();
});
