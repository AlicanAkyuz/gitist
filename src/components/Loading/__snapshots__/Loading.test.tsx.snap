// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading corrently renders a loading message 1`] = `
<p>
  YÃ¼kleniyor...
</p>
`;
